exporters/token/exporter.go

```diff
@@ -95,6 +95,12 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 		return
 	}
 
+	tokenRegex, err := regexp.Compile(`.\..{24}`)
```

Comment:
I remember discussing this before I left - what is missing to merge this?

Reply:
I had some concern about token formats. Vault docs show tokens as UUIDs, but I checked a few clusters and found them all in this format. I asked if anyone knew about this format discrepancy, but I guess no one did. I would like to investigate it some more before merging. Checking one recent TC on each installation and a few different provider CPs should be sufficient. I will try to wrap this up today so it can go in our next releases.

Reply:
I see now that TC nodes don't have vault tokens--only CP nodes. I checked two CPs for each provider and didn't find any tokens in UUID format so I'll go with this.